app/src/containers/WalletTransfer/WalletTransfer.vue

```diff
@@ -394,6 +395,13 @@ export default {
     }
   },
   computed: {
+    verifierOptions() {
+      const verifiers = JSON.parse(JSON.stringify(ALLOWED_VERIFIERS))
```

Comment:
Why stringify and parse.?

Reply:
This is for deep clone.. Cause i need to modify verifier labels with translation without affecting enums value.